sshcommand

```diff
@@ -58,6 +58,12 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=$NAME" "$USERHOME/.ssh/authorized_keys" || true)
```

Comment:
Unfortunately #15 doesn't work in the docker container. I'll get a patch in shortly.

@alessio (sad panda)
```
root@box223:/app# sshcommand create sshcommand_user ls
groupadd: group 'sshcommand_user' already exists
root@box223:/app# sed -n 's#^NAME="\(.*\)"#\1#p' /etc/os-release
Debian GNU/Linux
```